omnigibson/utils/bddl_utils.py

```diff
@@ -191,7 +191,7 @@ def process_single_condition(condition):
     "insource": ObjectStateInsourcePredicate,
 }
 
-KINEMATIC_STATES_BDDL = frozenset([state.__name__.lower() for state in _KINEMATIC_STATE_SET])
+KINEMATIC_STATES_BDDL = frozenset([state.__name__.lower() for state in _KINEMATIC_STATE_SET] + ["attached"])
```

Comment:
why is AttachedTo not a kinematic state?

Reply:
Not sure, but I don't want to change the current behavior. We can make it inherit KinematicStateMixin later.